### PR TITLE
feat: add filter options

### PR DESCRIPTION
**Pull request by barbara-chaves**
This PR adds the filter options to the discovery pages

## Testing instructions

To see the design on figma: [HeCo](https://www.figma.com/file/KC49uMR9t1YARmFR0Sr6aN/) -> Prototype -> Sign up -> page 22

It will be possible to filter the projects by:
* Category
* SDG
* Instrument
* Ticket size
* Verified content

Only one value per filter can be chosen

## Tracking

[LET-402](https://vizzuality.atlassian.net/browse/LET-402)
[LET-400](https://vizzuality.atlassian.net/browse/LET-400)

## Screenshots
Screenshot 2022-05-26 at 20 09 39
Screenshot 2022-05-26 at 20 09 26

### CODE REVIEW

**Comment by SARodrigues** [comment]
One thing that seems to be missing is displaying the selected on the search bar. Is it not part of this one? 

search_bar

**Comment by SARodrigues** [comment]
Another two issues I found visually: 

1. Display on mobile looks a bit wonky to me  
Screenshot 2022-05-24 at 08 46 06

2. The only way to close the filters search component seems to be to click on the _Filters_ button again. 
    This is not intuitive. I would expect it to close when I click outside the search bar / filters dropdown at least.

**Comment by barbara-chaves** [comment]
> One thing that seems to be missing is displaying the selected on the search bar. Is it not part of this one?
> 
> search_bar

You are right. I am going to include this

File: frontend/package.json [hunk 1]
```diff
@@ -82,6 +82,7 @@
     "jsonwebtoken": "^8.5.1",
     "lodash-es": "^4.17.21",
     "maplibre-gl": ">=1.15.2",
+    "material-ui-chip-input": "^1.1.0",
```
Comment (clementprdhomme) [inline frontend/package.json:85]: You're not using this, aren't you?

**Review by clementprdhomme — COMMENTED** [review]
A few more comments:
- The SDG filter doesn't seem to work: if I select a value and apply, the value is reset.
- The filters are reset/cleared when changing tab.
- On mobile the search bar looks broken (the header is also too tall I think).

The search bar doesn't have a nice look on mobile

File: frontend/components/forms/combobox/component.tsx [hunk 2]
```diff
@@ -92,6 +107,18 @@ export const Combobox = <FormValues extends FieldValues, T extends object>({
...
+        <button {...clearButtonProps} ref={clearRef} className="shrink-0" onClick={onClear}>
+          <Icon
+            aria-hidden={true}
+            icon={X}
+            className={cx({
+              'inline-block w-5 h-5 ml-2 text-gray-600 shrink-0 transition': true,
+              'rotate-180': state.isOpen,
+            })}
+          />
+        </button>
```
[the remark this answers is not in the document]
Comment (barbara-chaves) [reply frontend/components/forms/combobox/component.tsx:120]: I am sorry, I am not using the Combobox on filters anymore, so I will remove the changes I made on it. I forgot to undo the changes before.

**Comment by barbara-chaves** [comment]
@SARodrigues @clementprdhomme, what do you think about this design for  mobiles? Id it is not good, I will ask Andrea for help. 
Screenshot 2022-05-30 at 16 27 40

**Comment by clementprdhomme** [comment]
@barbara-chaves I think it would be better to ask Andreia, yes 😉.

**Comment by barbara-chaves** [comment]
> @barbara-chaves I think it would be better to ask Andreia, yes 😉.

Andrea will design the mobile version. A task to change the style will be added on the future.

**Review by SARodrigues — APPROVED** [review]

**Pull request merged**